### PR TITLE
pathway for setting up mariadb on mydb

### PR DESCRIPTION
**Pull request by atombaby**

### CODE REVIEW

**Comment by dtenenba** [comment]
I know this is probably beyond the scope of this PR, but the two names (mydb and db4sci) are really confusing. The web app is referred to in text as db4sci but the url and the db host is given as mydb. I know there is history behind the names, but can we settle on one of them? We could add a cname db4sci and just refer to it as that from now on and be done with it, leaving mydb for backwards compatibility.

Otherwise it looks good. I know the pathway is specific to mariadb but might want to mention the other database types that are available just in case someone lands on this page but really wants postgres or mongo or something.

**Comment by dtenenba** [comment]
PS dont' worry about the failing check - the runner seems to be having networking issues. It seems transient.

File: _pathways/path-mydb-mariadb.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+## Steps
+
+### Log in to db4sci
+
+Browse to https://mydb.fredhutch.org/login and enter your HutchNet ID
+
+### Create the Database Container
+
+Once logged into db4sci, click over to "Create DB Container" and select "MariaDB (MySQL)" from the list.  This will present a form with a number of fields you must fill in:
+
```
Comment (atombaby) [inline _pathways/path-mydb-mariadb.md:25]: Most everything (lines 26-35) in this section should probably be moved to a new doc on creating databases in mydb and linked to from this section.

**Comment by vortexing** [comment]
Edit names in this page too?  https://sciwiki.fredhutch.org/scicomputing/store_databases/

**Comment by atombaby** [comment]
- move store_db4sci into compdemos
 - reference said file from main database article in _sciomcomputing
 - reference pathway in scicomputing
 - change references to db4sci to mydb where it is our installed system

File: _pathways/path-mydb-mariadb.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+### Start a Session on a Rhino
+
+Open a shell on rhino with SSH or NoMachine.
+
+### Load the MariaDB Module
+
+Load the MariaDB module with the command `ml MariaDB`.
+
+### Connect to the Database
+
```
[the remark this answers is not in the document]
Comment (atombaby) [reply _pathways/path-mydb-mariadb.md:35]: Moved and linked

File: _pathways/path-mydb-mariadb.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+main_author: Michael Gutteridge
+primary_reviewers: atombaby, dtenenba, fizwit
+---
+
+This pathway will show you how to configure a MariaDB (MySQL) database on DB4Sci (aka MyDB) and connect to it using the command line tools on a rhino/gizmo node.
+
+## Pre-requisites
+A workstation, HutchNet ID, access to the internet and campus network, and browser
+
+If you are unfamiliar with any of these terms, hover over them to find more information.
```
[the remark this answers is not in the document]
Comment (atombaby) [reply _pathways/path-mydb-mariadb.md:12]: added in 87059977fa48c821176258a63695af1c06d1f4d6

File: _pathways/path-mydb-mariadb.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
+---
+title: Configure and Use a MariaDB Database on DB4Sci
+main_author: Michael Gutteridge
+primary_reviewers: atombaby, dtenenba, fizwit
+---
+
+This pathway will show you how to configure a MariaDB (MySQL) database on DB4Sci (aka MyDB) and connect to it using the command line tools on a rhino/gizmo node.
+
```
[the remark this answers is not in the document]
Comment (atombaby) [reply _pathways/path-mydb-mariadb.md:8]: Added note in ae086ca

**Comment by atombaby** [comment]
I think I am ready to merge this PR if someone can review and approve.

File: _pathways/path-mydb-mariadb.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+Browse to https://mydb.fredhutch.org/login and enter your HutchNet ID
+
+### Create the Database Container
+
+Follow the instructions [here](/compdemos/mydb#provisioning) to configure an instance of a MariaDB container.  Record the username, password, and port in a secure location.
+
+### Start a Session on a Rhino
+
+Open a shell on rhino with SSH or NoMachine.
```
Comment (atombaby) [inline _pathways/path-mydb-mariadb.md:32]: Add link to instructions for shell session here

File: _pathways/path-mydb-mariadb.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+### Load the MariaDB Module
+
+Load the MariaDB module with the command `ml MariaDB`.
```
Comment (atombaby) [inline _pathways/path-mydb-mariadb.md:36]: Add link to modules documentation

**Review by dtenenba — APPROVED** [review]
👍

File: _pathways/path-mydb-mariadb.md (hunk 5 above)
Comment (atombaby) [reply _pathways/path-mydb-mariadb.md]: See 5146b3667fedd06ebfa78345c602abbee18a71f1

File: _pathways/path-mydb-mariadb.md (hunk 6 above)
Comment (atombaby) [reply _pathways/path-mydb-mariadb.md]: see 5146b3667fedd06ebfa78345c602abbee18a71f1

**Pull request merged**